Makefile

```diff
@@ -39,3 +39,47 @@ buildenv:
 clean:
 	cargo clean
 .PHONY: clean
+# Publishes all crates to crates.io.
+#
+# The sleep call is needed because we've observed that crates are sometimes
+# not found by cargo immediately after a publish.  Sleeping on this is bad,
+# but there doesn't seem to be a much better option available.
+define publish
+	( cd $(1) && cargo publish && sleep 5)
+endef
+
+# This is not the best method, since it will error out if a crate has already
+# been published.
+.PHONY: publish
+publish:
+	$(call publish,rust_icu_sys)
+	$(call publish,rust_icu_common)
+	$(call publish,rust_icu_uenum)
+	$(call publish,rust_icu_ustring)
+	$(call publish,rust_icu_utext)
+	$(call publish,rust_icu_uloc)
+	$(call publish,rust_icu_ucal)
+	$(call publish,rust_icu_udat)
+	$(call publish,rust_icu_udata)
+
+# A helper to up-rev the cargo crate versions.
+UPREV_OLD_VERSION ?= 0.0.4
```

Comment:
Will this need to be adjusted after every version?

Reply:
Yes; it does not need to get adjusted every change to the build environment; unless there is something we really want to fix ASAP.